packages/console/src/components/Pagination/index.module.scss

```diff
@@ -0,0 +1,37 @@
+@use '@/scss/underscore' as _;
+
+.pagination {
+  display: flex;
+  justify-content: right;
+
+  li {
+    list-style: none;
+
+    &:not(:first-child) {
+      margin-left: _.unit(2);
+    }
+
+    .button {
+      border-radius: _.unit(1.5);
```

Comment:
nit: use `px` directly?

Reply:
The border-radius seems to be 4px only. Use 1 unit instead.

<img width="762" alt="image" src="https://user-images.githubusercontent.com/12833674/160319362-69e0275e-1a00-4f33-8b79-46e6ab6abdc6.png">

Reply:
<img width="526" alt="image" src="https://user-images.githubusercontent.com/5717882/160322431-c8ee4fe5-3cdd-4eda-8152-94c5f4b6ad81.png">

We follow the "Pagination" part in "Design System". Informed Guamian to fix that.

Reply:
updated to "6px", PTAL